edx_exams/apps/lti/views.py

```diff
@@ -101,14 +101,111 @@ def acs(request, lti_config_id):
         return Response(status=400)
 
     if action == 'flag':
+        # NOTE: The flag action is not yet supported.
+        # TODO: Make the flag action actually modify the exam attempt data (or perhaps another model?)
         success_msg = (
-            f'Flagging exam for user with id {anonymous_user_id} '
+            f'NOTE: The flag action is not yet supported. The following is a placeholder message.'
+            f'Flagging exam attempt for user with id {anonymous_user_id} '
             f'with resource id {resource_id} and attempt number {attempt_number} '
             f'for lti config id {lti_config_id}, status {attempt.status}, exam id {attempt.exam.id}, '
             f'and attempt id {attempt.id}.'
         )
         log.info(success_msg)
-    return Response(status=200)
+
+    elif action == 'terminate':
+        # Upon receiving a terminate request, the attempt referenced should have their status updated
+        # depending on the reason for termination (reason_code) and the incident_severity (scaling from 0.0 to 1.0).
+        # If the severity is greater than 0.25, then the attempt is marked for secondary review.
+
+        # Get the reason code for the termination and ensure it's a string to comply with the LTI specs
+        # See: http://www.imsglobal.org/spec/proctoring/v1p0#h.rsq8h6qxveab
+        reason_code_key = data['reason_code']
+        if not isinstance(reason_code_key, str):
+            error_msg = 'ERROR: reason_code must be passed to the ACS endpoint as a string per LTI specs.'
```

Comment:
might want to just change this to 'reason_code is required'. It's optional to implement by the spec but edx does require it.